sydent/http/servlets/threepidbindservlet.py

```diff
@@ -63,15 +62,19 @@ def render_POST(self, request):
             valSessionStore = ThreePidValSessionStore(self.sydent)
             s = valSessionStore.getValidatedSession(sid, clientSecret)
         except IncorrectClientSecretException:
-            return noMatchError
+            raise noMatchError
```

Comment:
Can we merge this and `InvalidSessionIdException` block and then we can move the creation of the exception object into here? (I don't know if the line numbers in stack trace come from where we raise or where we create an exception off the top of my head)

Reply:
Ah, good point. You can catch two exceptions at once. 👍